chemprop/nn/predictors.py

```diff
@@ -144,8 +151,8 @@ def encode(self, Z: Tensor, i: int) -> Tensor:
 @PredictorRegistry.register("regression")
 class RegressionFFN(_FFNPredictorBase):
     n_targets = 1
-    _default_criterion = MSELoss()
-    _default_metric = MSEMetric()
+    _default_criterion = MSELoss
+    _default_metric = MSEMetric
```

Comment:
one minor comment before merging. These should all be renamed:
```suggestion
    _T_default_criterion = MSELoss
    _T_default_metric = MSEMetric
```
because they're all `type`s now rather than `object`s. (ignoring the semantic of Python that even `type` is an `object`)

Reply:
Thanks, didn't know that naming convention